WatchDogCore/WatchDog/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,12 @@
+eclipse.preferences.version=1
```

Comment:
I think this file is not needed. In fact, you can ignore the whole .settings folder